### DISCUSSION TITLE
quering an expression database

### DISCUSSION THREAD

**Discussion by zt1947** [Open-Systems-Pharmacology/Forum#1414] [Questions & Problems]
**IMPORTANT: Issues are now obsolete**
Starting May 7th, 2021, the OSP issues will not be used anymore for discussions. Instead, the Discussions feature from Github will be used

Please see the announcement here:
https://github.com/Open-Systems-Pharmacology/Forum/issues/894

To start a discussion, simply go to 
https://github.com/Open-Systems-Pharmacology/Forum/discussions and click on New Discussion

**Comment by zt1947** [comment]
In a new project, I am trying to define the metabolizing enzymes relevant to the compound involved. I did download the human expression database and defined its location under Applications in Options. However when I right click on "Metabolizing Enzymes" and select  "Add Enzyme", only the manual addition option opens while the option (Database query) does not show up. Enclosed is a copy of the screen illustrating the above.
[Doc1.docx](https://github.com/Open-Systems-Pharmacology/Forum/files/10260806/Doc1.docx)
I would appreciate help in resolving this issue

**Comment by Yuri05** [comment]
Please see the answer to your question in this recent discusion: https://github.com/Open-Systems-Pharmacology/Forum/discussions/1359
